src/__tests__/SelectMenu.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react'
+import SelectMenu from '../SelectMenu'
+import Button from '../Button'
+import {mount} from '../utils/testing'
+import {COMMON} from '../constants'
+import {render as HTMLRender, cleanup} from '@testing-library/react'
+import {axe, toHaveNoViolations} from 'jest-axe'
+import 'babel-polyfill'
+expect.extend(toHaveNoViolations)
+
+const BasicSelectMenu = () => {
+  return (
+    <SelectMenu>
+      <Button as="summary">Projects</Button>
```

Comment:
Have you tested this paradigm in TypeScript? I remember some weirdness with the `as` prop existing.

Reply:
Tested this out in code sandbox and it looks like it works fine: https://codesandbox.io/s/primer-components-ts-testing-47dbw (checkout the SelectMenuTest file)

Reply:
Looks good to me!